app/src/main/res/layout/hints_summary.xml

```diff
@@ -94,7 +95,7 @@
             android:layout_marginTop="8dp"
             android:layout_marginEnd="8dp"
             android:layout_marginBottom="4dp"
-            android:contentDescription="@{viewModel.computeHintListDropDownIconContentDescription()}"
```

Comment:
One more suggestion, 

use `ImageButton` instead of `ImageView`, so that the talkback will correctly read the output for button instead of `Image`

Also, why was the content description should be optimised, i.e.,
- if list is expanded, 'Hide hint'
- if list is not-expanded, 'Show hint', 

**Also, note, I think similar changes needs to be done for `Solution-summary` too, please check that too..**

Reply:
I have made required changes for Solution-summary. Updated changes and video in PR's description. Thanks